quirks/change-password-URLs.json

```diff
@@ -4,5 +4,14 @@
     "ea.com": "https://myaccount.ea.com/cp-ui/security/index",
     "twitter.com": "https://twitter.com/settings/password",
     "fnac.com": "https://secure.fnac.com/account/update-password",
-    "microsoft.com": "https://account.live.com/password/Change"
+    "microsoft.com": "https://account.live.com/password/Change",
+    "upc.cz": "https://my.upc.cz/ForgottenPwd/Choice",
+    "linkedin.com": "https://www.linkedin.com/checkpoint/rp/request-password-reset",
+    "wikipedia.org": "https://en.wikipedia.org/wiki/Special:PasswordReset",
+    "google.com": "https://www.google.com/accounts/ForgotPasswd",
```

Comment:
I'm not 100% sure but. Its this file suppose to hold password change URLs ?

Reply:
That’s right! Change, not forgot.